packages/shorebird_cli/lib/src/commands/patch/android_patcher.dart

```diff
@@ -206,21 +203,8 @@ Looked in:
   }
 
   @override
-  Future<CreatePatchMetadata> createPatchMetadata(DiffStatus diffStatus) async {
-    return CreatePatchMetadata(
-      releasePlatform: releaseType.releasePlatform,
-      usedIgnoreAssetChangesFlag: allowAssetDiffs,
-      hasAssetChanges: diffStatus.hasAssetChanges,
-      usedIgnoreNativeChangesFlag: allowNativeDiffs,
-      hasNativeChanges: diffStatus.hasNativeChanges,
-      linkPercentage: null,
-      environment: BuildEnvironmentMetadata(
-        flutterRevision: shorebirdEnv.flutterRevision,
-        operatingSystem: platform.operatingSystem,
-        operatingSystemVersion: platform.operatingSystemVersion,
-        shorebirdVersion: packageVersion,
-        xcodeVersion: null,
-      ),
-    );
-  }
+  Future<CreatePatchMetadata> updatedCreatePatchMetadata(
```

Comment:
minor but we can make this the default implementation so that patchers and releasers that don't want to modify the metadata don't even need to override this API.

Reply:
Done